tldraw/tldraw
# feat(blog): Hook up newsletter form with SendGrid

### Change type


### Test plan

1. Submit form with email address
2. Check the SendGrid list

### To do

- [ ] Add SendGrid list id as an environment variable
